docs/guide/TNGF/tngfue-installation.md

```diff
@@ -57,17 +63,33 @@ cd tngfue
 
 ### TNGFUE Configuration
 
```

Comment:
Hello, sure.

When I wrote this section, I tried to follow the design from [this section](https://free5gc.org/guide/3-install-free5gc/#b-install-control-plane-elements) of free5GC's install page.

Do you think it could be better if we move the "manual" steps (e.g. To configure and build manually) to another separate section, let's say "section 4.B." and let the prepare script to be alone in 4.A. ?

Reply:
Sounds great!

Reply:
First of all, thanks for your review and comments. I've applied the suggested changes in https://github.com/free5gc/free5gc.github.io/pull/164/commits/0c88c0e0cf3a0d4cacc275650b961f046b08d1ba